packages/react/src/useField.js

```diff
@@ -256,6 +263,16 @@ const useField = ({
           value
           === ''
         )
+        && (
+          (
+            inputValue
+            == null
+          )
+          || (
+            inputValue
+            === ''
+          )
```

Comment:
I think the `''` check should come before the `null`-`undefined` check.

I say that because we're doing `value === ''` the line above. When I read this in the diff, it was difficult to understand the relation between the two. Having the `''` checks ordered first would make it more clear what's going on without having to think about it.